### PR TITLE
set mode only once in operator roles upgrade

### PR DESCRIPTION
**Pull request by ydayagi**
[sda-6745](https://issues.redhat.com//browse/sda-6745)

Signed-off-by: Yaron Dayagi <ydayagi@redhat.com>

### CODE REVIEW

File: cmd/upgrade/operatorroles/cmd.go [hunk 1]
```diff
@@ -214,12 +214,13 @@ func run(cmd *cobra.Command, argv []string) error {
 		os.Exit(1)
 	}
 
+	mode, err = handleModeFlag(cmd, skipInteractive, mode, err, r.Reporter)
```
Comment (oriAdler) [inline cmd/upgrade/operatorroles/cmd.go:217]: Do we need to wrap it with the following condition?
`if isOperatorPolicyUpgradeNeeded || len(missingRolesInCS) > 0 {`

Comment (ydayagi) [reply cmd/upgrade/operatorroles/cmd.go:217]: the condition is already checked earlier and we return if it is not satisfied

Comment (oriAdler) [reply cmd/upgrade/operatorroles/cmd.go:217]: Oh right, missed that, thanks.

**Pull request merged**